src/app/shared/components/template/components/text-bubble/text-bubble.component.ts

```diff
@@ -8,7 +8,9 @@ interface ITextBubbleParams {
   /** TEMPLATE PARAMETER: "speaker_position". The position of the speaker image and speech bubble tail */
   speakerPosition: "left" | "right";
   /** TEMPLATE PARAMETER: "variant" */
-  variant: "gray" | "primary" | "secondary" | "no-border";
+  variant: "gray" | "primary" | "secondary" | "no-border" | "orange" | "yellow";
+  /** TEMPLATE PARAMETER: "speaker_name". The name of the speaker */
```

Comment:
Good to have this comment in line with our documentation convention. It would also be good to explicitly list the new parameter in the PR description, that way @esmeetewinkel can functionally test the new functionality without needing to engage with the code, plus the PR can act as a form of documentation for future reference.

The same goes for the new variants: it would be good to explicitly lit them in the PR description

Reply:
The PR description has been updated to list the new parameters @jfmcquade